### ISSUE TITLE
Rogue: Blade Flurry

### ISSUE THREAD

**Issue by elorix** [dalaranwow/dalaran-wow#1383]
Ok so basically the idea behind this ability is to transfer 100% of the damage you deal to target x onto target y. The damage you deal to target x is being reduced by armor, but the damage you deal to target y with blade flurry is being reduced again by armor, and therefore gets double armor penalty. This shouldnt be the case, the transferred damage should do exactly as much and bypass armor.

**Comment by Klala** [comment]
In that case, why this issue is still open?

**Comment by marsalandrei** [comment]
Because it is still bugged. Can confirm it. Test it by attacking two of the same mobs and of the same levels. Being 2 of the same mob and level they will have the same armor. Say they both have 50% reduction from armor. I do 1k damage and it will hit my target for 500, but it will do 250 on the second, because it will be affected by both targets armor which shouldn't happen. It should deal 500 on the second target too, since it has 50% too. As is now, the damage dealt to the second target is firstly reduced by the armor of the main target, then by it's own armor, so you will deal only 75% of that 1k to the second target

**Comment by robinsch** [comment]
On PTR.

**Comment by Crispybuilt** [comment]
Tested on PTR, it appears it works fine now.

**Comment by Edzero** [comment]
> Blade Flurry should not mirror the exact amount of damage, "bypassing" armor as you state.
The amount of damage taken by the secondary target should be mitigated by its armor, the same way a normal attack against it would.

For example:

Target X has 10% armor reduction (primary target)
Target Y has 80% armor reduction (secondary target)
If Blade Flurry is active and I deal 1000 damage to Target X (crit strike or not, shouldn't matter), which is mitigated by its armor to a final value of 900 (10% armor reduction). The extra attack given by Blade Flurry should deal 200 damage to Target Y (80% armor reduction).
That being said, if you deal 15k damage to a completely naked player, you will not be dealing 15k to a fully plate geared player next to him..

To sum it up, Blade Flurry formula should re-calculate the armor of the secondary target.

This is not working correctly on the PTR.

I lost the log due to a crash but the result is certain. If X has lower armor and is attacked and is dealt 900 damage as given in the example above, Y will also take 900 damage. If Y is attacked, he will take 200 damage and X will also take 200 damage.

**Comment by Rushor** [comment]
yes because the spell got the flagextra to ignore armor.

i think that was meaned by "That being said, if you deal 15k damage to a completely naked player, you will not be dealing 15k to a fully plate geared player next to him.."

**Comment by Rushor** [comment]
However the fix is a good step in the right direction and improves the gameplay. i think it should be applied on the changelog for now

**Comment by InshadowsAlgalon** [comment]
Also if u watch this video of LK 25hc world 1st https://www.youtube.com/watch?v=RSeVCw3hVgQ , from a rogue perspective on 5:37 he pops BF+KS, and u can see that BF duplicates exact same hits of other abilities and melee. 
That being said, i doubt he has 100% armor pen, because he uses Herkumi War Token, and they were trying for worlds 1st, so no one had best gear. And now on Beta ICC test, rogue struggles to get 100% arp without STS, and he was using Rupture (which full rogues dont use) but yet again they had Expose Armor on LK.
Considering all your previous comments about armor of secondary target, they most certainly didnt have armor reduction on adds, and rogue didnt have 100% armor reduction on adds, so how did BF hit exact same numbers on adds as it hit on boss?
Fukk, he was hiting Raging Spirit. Still i think it was usefull observations. Usefull rogue moments are 5:37 , 8:05, 13:41, 19:54
13:41 shows that melee hits while in Killing Spree is active, so those melee hits can generate Combat Potency while in KS.

**Comment by Edzero** [comment]
It should be that armor is taken into account separately for both targets.

As Wonopon said, it was changed in 1.10.0 and remained the same throughout WotLK.

http://wowwiki.wikia.com/wiki/Blade_Flurry

As discussed previously, this is not working 100% properly on the PTR but since the fix is an improvement it will be pushed to the next changelog and polished later.

**Comment by TeridanDW** [comment]
The original issue posted by @elorix is fixed on ptr (2x armor reduction on 2-nd target), however:

> That being said, if you deal 15k damage to a completely naked player, you will not be dealing 15k to a fully plate geared player next to him..

is still the case.  Will go with @Rushor 's word here and say:

> However the fix is a good step in the right direction and improves the gameplay. i think it should be applied on the changelog for now.
